### ISSUE TITLE
Qukeys rollover improvement idea

### ISSUE THREAD

**Issue by gedankenexperimenter** [keyboardio/Kaleidoscope#925]
I just had an idea that might improve the accuracy of Qukeys in guessing user intentions when there's rollover from a qukey to a subsequent key.

My hypothesis is that users are much less likely to intend the qukey press as a “hold” if the qukey's press follows the _previous_ key's press closely. Additionally, if there's rollover from a non-modifier (maybe whitespace, too?) key to the qukey, it's also less likely that a “hold” was intended.

Basically, we could add a requirement to have a minimum delay between key presses and/or a maximum overlap from the previous key (depending on key value). Iff the requirement is fulfilled, the qukey becomes eligible to take on its alternate value.

As a bonus, it could reduce delays in flushing certain events from the queue.
